1/test/run

```diff
@@ -137,7 +137,46 @@ function run_tests() {
   echo "  Success!"
 }
 
+function run_s2i_tests() {
+  if [ "$(which s2i)" == "" ]; then
+    echo "WARNING: Please install the s2i binary to run the s2i tests."
+    return
+  fi
+  output_image="openshift/s2i-test-jenkins"
+  s2i build --context-dir="1/test/s2i" --loglevel=3 --pull-policy=never \
+    file://$(pwd)/.. $IMAGE_NAME $output_image
+  IMAGE_NAME="${output_image}" create_container s2i-jenkins-test $VOLUME
+  cid=s2i-jenkins-test
+  CONTAINER_IP=$(get_container_ip ${cid})
+  test_connection ${cid}
+  set +e
+  if ! docker exec ${cid} ls /var/lib/jenkins/plugins/sample.jpi.pinned; then
+    echo "Fail: The sample.jpi.pinned plugin was not copied"
+    return 1
+  fi
+  if ! docker exec ${cid} ls /var/lib/jenkins/plugins/ansicolor.jpi; then
+    echo "Fail: The ansicolor plugin should be installed"
+    return 1
+  fi
+  if ! docker exec ${cid} ls /var/lib/jenkins/jobs/sample-app-test/config.xml; then
+    echo "Fail: The sample-app-test job was not copied"
+    return 1
+  fi
+  docker exec ${cid} cat /var/lib/jenkins/config.xml | grep -q "s2i-test-config"
+  if [ "$?" != "0" ]; then
+    echo "Fail: The config.xml was not copied"
+    return 1
+  fi
+  set -e
+  test_get_job admin password sample-app-test
+  test_create_job admin password testJob
```

Comment:
it's confusing that the job your config files provide is the same name as the job the other test creates/gets.  can you rename one of them?